content/md/first-steps/manage-data-quality.md

```diff
@@ -48,3 +45,36 @@ In the data quality panel of the product edit form, you can see which attribute
 
 ![Image data quality attribute groups reminder](../img/data-quality-attribute-groups-reminder.png)
 :::
+
+# Manage your dictionary
+Either because your product names are unique or due to industry acronyms that are not part of regular dictionaries, you can add words in your personal dictionary to fit your special needs and avoid miscalculation of your Quality Score.
```

Comment:
Can we add a link on the "Quality score" words to link to the article that talks about it?